lib/sphero.js

```diff
@@ -24,6 +24,16 @@ function classCallCheck(instance, Constructor) {
   }
 }
 
+/**
+ * Creates a new sphero instance
+ * @constructor
+ * @param {String} address of the connected sphero
+ * @param {Object} opts for sphero setup
+ * @example
+ * var sphero = require("sphero");
+ * var spheroRPB = sphero("/dev/rfcomm0", { timeout: 300 });
```

Comment:
Is there a place we describe the typical formats of port naming for OSX, Windows, and Linux? Having the Linux port as the example is probably the smallest audience of users, I suggest using OSX as default.